src/lib/internal.h

```diff
@@ -622,7 +627,7 @@ ns_to_timespec(uint64_t ns, struct timespec* ts){
 
 static inline void
 cell_debug(const egcpool* p, const nccell* c){
-  fprintf(stderr, "gcluster: %u %s style: 0x%04x chan: 0x%016jx\n",
+  fprintf(stderr, "gcluster: %u %s style: 0x%04x chan: 0x%016llx\n",
```

Comment:
i seem to recall this drawing a warning somewhere when using a uint64_t, though i can't recall why or where currently. i usually just %jx along with an explicit cast to `uintmax_t`. but if this works, it works.

Reply:
Yeah, there was a crapload of warnings, annoyed the hell out of me... :)